tools/ncbi_blast_plus/ncbi_macros.xml

```diff
@@ -1,13 +1,14 @@
 <macros>
-    <token name="@WRAPPER_VERSION@">0.3.3</token>
+    <token name="@WRAPPER_VERSION@">0.3.4</token>
     <xml name="parallelism">
         <!-- If job splitting is enabled, break up the query file into parts -->
         <parallelism method="multi" split_inputs="query" split_mode="to_size" split_size="1000" merge_outputs="output1" />
     </xml>
 
     <xml name="preamble">
         <requirements>
-            <requirement type="package" version="2.7.1">blast</requirement>
+            <requirement type="package" version="2.9.0">blast</requirement>
+            <requirement type="package" version="13.3">entrez-direct</requirement>
```

Comment:
Does BLAST need Entrez Direct for the taxonomy look up?

Reply:
Yes, unfortunately. The script ships with blast itself, but relies on esearch and friends. 

> Note: this script depends on EDirect and it only works on Linux and Mac

via https://www.ncbi.nlm.nih.gov/books/NBK546209/